assets/steane_qrisp.py

```diff
@@ -0,0 +1,48 @@
+# The following is how you would generate a Steane code using Qrisp.
```

Comment:
Please add some comments along the lines of this file is used by so and so notebook in the documentation.